src/components/AdminDashboard.vue

```diff
@@ -204,44 +207,41 @@ import graph_style from "../assets/graph-style.json";
 // The default graph representation
 // TODO: This file will be deleted and the graph will be stored on the DB
 import json from "../assets/nodes.json";
+// The popup component for database logic information
+import PopUp from './PopUp.vue'
+
+  export default {
+    name: 'admin-dashboard',
+    props: ['isAdmin'],
```

Comment:
I think props should be the code below unless I'm missing something. 

props: ["isAdmin", "Nodes", "graph-status", "graph-id", "Cords"],

Reply:
Looks like it's been changed since I made the PR.

Reply:
I'll pull down master, add the button and make a new pull request if that works for you.

Reply:
Merging master into database_logic_button should do the trick rather than having to make a new pr